internal/kafka/kafka.go

```diff
@@ -11,6 +11,7 @@ import (
 
 	config "github.com/redhatinsights/payload-tracker-go/internal/config"
 	l "github.com/redhatinsights/payload-tracker-go/internal/logging"
+	"github.com/redhatinsights/payload-tracker-go/internal/endpoints"
```

Comment:
Same as above.  I don't think the endpoints package should be used by the kafka package.  It feels like there is some misplaced responsibilities.